api/provider/aws/deploy_list.go

```diff
@@ -34,8 +36,12 @@ func (d *DeployProvider) listTaskDefinitionARNs() ([]string, error) {
 	familyPrefix := addLayer0Prefix(d.Config.Instance(), "")
 	input := &ecs.ListTaskDefinitionFamiliesInput{}
 	input.SetFamilyPrefix(familyPrefix)
-	// TODO: Revisit how Inactive and Active Task Definitions might want to be returned to the client
-	input.SetStatus(ecs.TaskDefinitionFamilyStatusActive)
+	if all {
```

Comment:
[nit] try to void if/else blocks. Set status active and then only have the “if” block for when all=true